server/server.go

```diff
@@ -430,7 +430,12 @@ func (s *Server) releaseContainerName(name string) {
 
 // Shutdown attempts to shut down the server's storage cleanly
 func (s *Server) Shutdown() error {
-	_, err := s.store.Shutdown(false)
+	_, err := os.Stat("/var/run/ocid.shutdown")
```

Comment:
I think we should replicate this whole logic on startup as well if we find this file. That way if machine rebooted before ocid has a chance to cleanup then it can do it on startup.